force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.html

```diff
@@ -129,36 +129,36 @@
                             <div class={rowClasses} style={fixedWidth} title={labels.lblStatus}>
                                 {item.status}</div>
                         </td>
-                        <td key={item.firstDonation} data-id={item.id} class={tdClasses} data-label={labels.firstDonation}>
+                        <td key={item.firstDonation} data-id={item.recurringDonation.Id} class={tdClasses} data-label={labels.firstDonation}>
                             <div class={rowClasses} style={fixedWidth} title={labels.firstDonation}>
                                 {item.firstDonation}
                             </div>
                         </td>
-                        <td key={item.recurringDonation.npe03__Last_Payment_Date__c} data-id={item.id} class={tdClasses} data-label={labels.mostRecentDonation}>
+                        <td key={item.mostRecentDonation} data-id={item.recurringDonation.Id} class={tdClasses} data-label={labels.mostRecentDonation}>
                             <div class={rowClasses} style={fixedWidth} title={labels.mostRecentDonation}>
                                 {item.recurringDonation.npe03__Last_Payment_Date__c}</div>
                         </td>
-                        <td key={item.nextDonation} data-id={item.id} class={tdClasses}  data-label={labels.nextDonation}>
+                        <td key={item.nextDonation} data-id={item.recurringDonation.Id} class={tdClasses}  data-label={labels.nextDonation}>
                             <div class={rowClasses} style={fixedWidth}> 
                                 {item.nexDonationFormatFirstElement} </br>
                                 {item.nexDonationFormatSecondElement}
                             </div>
                         </td>
-                        <td key={item.lastModified} data-id={item.id} class={tdClasses}  data-label={labels.lastModified}>
+                        <td key={item.lastModified} data-id={item.recurringDonation.Id} class={tdClasses}  data-label={labels.lastModified}>
                             <div class={rowClasses} style={fixedWidth} title="Last Modified">
```

Comment:
hardcoded text (though maybe this was fixed in another PR?)

Reply:
Yes, this was fixed in another PR, would we still need to fix it here?

Reply:
If it's a small change then I recommend it, especially if this is going to be UI tested before merging. It also ensures that future merges don't accidentally overwrite the fix to the label with hardcoded text.

Reply:
Fixed!!